lektor/assets.py

```diff
@@ -87,7 +87,7 @@ def resolve_url_path(self, url_path):
         return None
 
     def __repr__(self):
-        return "<%s %r>" % (
+        return "<{} {!r}>".format(
```

Comment:
There are a number of these cases where `%`-formatted strings are converted to `.format()` formatted strings rather than f-strings.

The `pyupgrade` README does [say](https://github.com/asottile/pyupgrade#f-strings):

> note: pyupgrade is intentionally timid and will not create an f-string if it would make the expression longer or if the substitution parameters are sufficiently complicated (as this can decrease readability).

That said, a lot of these cases seem like they are short enough that they would, in fact, be cleaner as f-strings.  Should we manually convert them?